tests/xrt/56_xclbin/main.cpp

```diff
@@ -0,0 +1,233 @@
+/**
+ * Copyright (C) 2016-2020 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <iostream>
+#include <stdexcept>
+#include <string>
+#include <cstring>
+
+// XRT includes
+#include "experimental/xrt_xclbin.h"
+#include "experimental/xrt_device.h"
+#include "experimental/xrt_kernel.h"
+#include "experimental/xrt_bo.h"
+
+// This value is shared with worgroup size in kernel.cl
+static const int COUNT = 1024;
+
+static const int C_SUCCESS = 0;
+
+static void usage()
```

Comment:
Modern C++ prefer anonymous namespace for all these `static` functions

Reply:
ok, will keep this in mind. This test was mostly a copy and paste job, just plugged in our new API. Maybe in future PR, I can help improve the tests across the board.

Reply:
Please contribute to https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-unnamed2 ;-)